layouts/private-wrapper.tsx

```diff
@@ -2,6 +2,7 @@ import { useUser } from "@supabase/auth-helpers-react";
 import { authSession } from "lib/hooks/authSession";
 import { useRouter } from "next/router";
 import React, { useEffect } from "react";
+import useStore from "lib/store";
```

Comment:
Why do you need the `store` here, I think we don't need it here right now. Could you remove it ??